specification/sql/resource-manager/Microsoft.Sql/2014-04-01/queries.json

```diff
@@ -0,0 +1,347 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Azure SQL Database",
+    "description": "Provides create, read, update and delete functionality for Azure SQL Database resources including servers, databases, elastic pools, recommendations, operations, and usage metrics.",
+    "version": "2014-04-01"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/servers/{serverName}/databases/{databaseName}/topQueries": {
+      "get": {
+        "tags": [
+          "Queries"
+        ],
+        "operationId": "Queries_ListByDatabase",
+        "description": "Gets a list of top queries by database.",
+        "x-ms-examples":{
+          "List top queries": { "$ref": "./examples/QueriesList.json" }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupParameter"
+          },
+          {
+            "$ref": "#/parameters/ServerNameParameter"
+          },
+          {
+            "$ref": "#/parameters/DatabaseNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/TopQueriesListResult"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": null
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Sql/topQueries/{queryId}/statistics": {
+      "get": {
+        "tags": [
+          "Queries"
+        ],
+        "operationId": "QueryStatistics_ListByQuery",
+        "description": "Lists a query's statistics.",
+        "x-ms-examples":{
+          "Get a query": { "$ref": "./examples/QueryStatisticsList.json" }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ResourceGroupParameter"
+          },
+          {
+            "$ref": "#/parameters/ServerNameParameter"
+          },
+          {
+            "$ref": "#/parameters/DatabaseNameParameter"
+          },
+          {
+            "name": "queryId",
+            "type": "string",
+            "description": "The id of the query",
+            "required": "true",
```

Comment:
looks like "required" value is in quotes, but should be a boolean - "required": true
CI still fails due to this, you can use the schema at https://raw.githubusercontent.com/Azure/autorest/master/schema/swagger-extensions.json to validate the file before submission.